index.js

```diff
@@ -32,8 +32,8 @@ module.exports = customService(async function index(service) {
     useUnifiedTopology: false,
   })
 
-  service.addRawCustomPlugin('GET', '/greetings', async function handler(req, reply) {
-    const currentUserId = req.getUserId()
+  service.addRawCustomPlugin('GET', '/greetings/:userId', async function handler(req, reply) {
```

Comment:
That's not the userId, userId are provided via headers.

Also the API is not clear, let's use a query string to apply a filter on the GET